src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const population = document.querySelectorAll('.population');
+const populationLength = Object.keys(population).length;
+console.log(population);
```

Comment:
remove console.log

Fix demo page (redeploy solution) - ask for help in the chat if you need